nebula/ui/components/inAppAuth/MZInAppAuthenticationInputs.qml

```diff
@@ -56,7 +56,7 @@ ColumnLayout {
                 Layout.fillWidth: true
                 _placeholderText: _inputPlaceholderText
                 Keys.onReturnPressed: col.submitInfo(textInput)
-                onDisplayTextChanged: if (hasError) hasError = false
+                onTextChanged: if (hasError) hasError = false
             }
```

Comment:
Interesting - I would expect these two signal handlers to be the same thing in this case. Do you know why this happens? Qt bug?

Reply:
I would too and it seems that they are handled the same in all cases apart from on ios. I'm not sure if this is a Qt "nuance" or something exacerbated by our[ special ios focus handling ](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/c647922f806b8b188b25e0d6163039fa689cca67/src/apps/vpn/ui/main.qml#L117).